Algorithm.Framework/Alphas/DualThrustAlphaModel.cs

```diff
@@ -0,0 +1,220 @@
+﻿using QuantConnect.Data;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Market;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Indicators;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```

Comment:
Some of these could be removed I think

Reply:
Please follow this pattern the Algorithm.Language folder has algorithms; see following for examples:
https://github.com/QuantConnect/Lean/pull/2932
https://github.com/QuantConnect/Lean/pull/2931
https://github.com/QuantConnect/Lean/pull/2930